### ISSUE TITLE
[Quest] [Borean Tundra] Leading the Ancestors Home

### ISSUE THREAD

**Issue by UrzaPlaneswalker** [holydeew/WoWHellgarve-Legion#134]
On Quest 
https://www.wowhead.com/wotlk/quest=11610/leading-the-ancestors-home

its not possible to complete https://www.wowhead.com/wotlk/npc=25398/elder-saganis-ceremony-completed

**Comment by AlveronAlsv** [comment]
The NPC phase has been changed from 2 to 1. The problem has been fixed (possibly permanent).

**Comment by skeezerbean** [comment]
With updating phase/flags as a GM will be permanent on this server unless changed by a dev/GM again. There are no SPP "updates" that get applied from official SPP, only the updates from myself or other devs, so the world DB won't get overwritten.

I'm guessing it was phaseMask that was changed. Depending on how this quest should be setup, it may need changed back some day but I've not done much in Borean. Mostly I fixed a lot in Howling Fjord. As long as it works ok now then feel free to close the issue for the time being and eventually it should get cleaned up a bit more if needed.
